weave

```diff
@@ -806,6 +806,19 @@ dns_args() {
     fi
 }
 
+rewrite_hosts() {
+    hosts_path_and_name=$(docker inspect -f '{{.HostsPath}} {{.Config.Hostname}}' $CONTAINER)
+    hosts=$(echo $hosts_path_and_name | cut -f1 -d" ")
+    name=$(echo $hosts_path_and_name | cut -f2 -d" ")
+    containers_path="$(dirname $(dirname "$hosts"))"
```

Comment:
>  so `/containers` wouldn't contain containers anymore but only the current container

That is actually very desirable. Unlike the proxy, we only need access to the dir of one container.

Obviously we should change the dir name to `/container`.